app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -237,6 +239,12 @@ class HomeFragment : Fragment() {
             )
         }
 
+        val applicationContext = requireContext().applicationContext
+        lifecycleScope.launch(Dispatchers.IO) {
+            val stories = PocketStoriesUseCases().GetPocketStories(applicationContext).invoke()
+            homeFragmentStore.dispatch(HomeFragmentAction.PocketArticlesChange(stories))
```

Comment:
I think we also tend to wrap these up into a feature and you can also hide it behind the FeatureFlags.

Reply:
Will do. Thanks!